rules/REGULA_R00004.rego

```diff
@@ -0,0 +1,21 @@
+package rules.REGULA_R00004
+
+resource_type = "aws_security_group"
+controls = {"CIS_4-1", "REGULA_R00004"}
+
+# VPC security groups should not permit unrestricted access from the internet to port 22 (SSH). Removing unfettered connectivity to remote console services, such as SSH, reduces a server's exposure to risk.
+
+invalid_port = 22
+
+invalid_ingress_block(block) {
+  block.cidr_blocks[_] == "0.0.0.0/0"
+  block.from_port <= invalid_port
+  block.to_port >= invalid_port
+}
+
+default allow = false
+
+allow {
+  block = input.ingress[_]
+  not invalid_ingress_block(block)
+}
```

Comment:
This should be a deny rule.  Currently it will pass if _one_ of the blocks is valid on line 20.